gamedb.py

```diff
@@ -64358,7 +64360,10 @@
     '6cd9bfe61a3bfbf7b3039d3a3743af6f': {
         'id': 'TTGM00000',
     },
-    '48615d361ff1026873ef5a1161a75e36': {
```

Comment:
Don't remove this,   I had a version of TTGM confirmed with this md5sum

Reply:
The one that starts with 48615 is my bad dump, I checked just in case with both pop-fe and bchunk.
The other, starting with 6cd9b, was there beforehand, so I assume that's your confirmed version. Sorry for any confusion.

Reply:
ah ok.